### ISSUE TITLE
UI - Fix Room text shift on iPhones

### ISSUE THREAD

**Issue by heatsink** [mobileappdevhm/dev-team-1-cie-app-in-flutter#225]
<img width="276" alt="screen shot 2018-06-24 at 3 25 05 pm" src="https://user-images.githubusercontent.com/9795754/41819579-efe861d8-77c2-11e8-8ec8-1dc1286740f7.png">
I believe this issue doesn't exist on Android, but on iphones the text seems to shift over and cause the Room text to be misaligned. 

A quick possible solution could be to delete the day text since it is already displayed in Red by the accordion, and move the room text to that area.

**Comment by jusnelda** [comment]
I don't have this issue on my iPhone (SE)

**Comment by RedTo** [comment]
![screenshot_1529928654](https://user-images.githubusercontent.com/15648487/41849560-a1ccaba2-7881-11e8-937d-c6c935d080aa.png)

This view looks like this on Android... It seems, that it is completely different?!

**Comment by RedTo** [comment]
@heatsink Do you still have this problem on your phone?

**Comment by heatsink** [comment]
Yes, I still have this issue on my phone. However, if no one else has this issue it could just be an issue with my IOS tools. 

If that's the case, then we can close this,
